src/js/utils/border.js

```diff
@@ -99,6 +118,19 @@ export const borderStyle = (borderData, responsive, theme) => {
         styles.push(breakpointStyle(breakpoint, responsiveStyle));
       }
     }
+    if (data.image) {
```

Comment:
Looks like this makes the border image silently supported on Grid, Table and Tag also (although without the proptypes and typescript support?)

Reply:
well Tag adds on top of Box wouldnt the `proptypes` and `typescript` be there from Box?

Reply:
I can add support for Grid,Table etc and look where else we are importing `borderStyle`

Reply:
I did not add `border` to propTypes within Table we didnt even have the `border` in there before, I know there are places in grommet where under the hood certain functionality works but we don't always have it in TS and PropTypes.